src/harness/testcases/WINNF_FT_S_HBT_testcase.py

```diff
@@ -0,0 +1,172 @@
+#    Copyright 2016 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+from datetime import datetime
+import json
+import os
+import unittest
+import sas
+from util import winnforum_testcase
+
+
+class HeartbeatTestcase(unittest.TestCase):
+
+  def setUp(self):
+    self._sas, self._sas_admin = sas.GetTestingSas()
+    self._sas_admin.Reset()
+
+  def tearDown(self):
+    pass
+
+  @winnforum_testcase
+  def test_WINNF_FT_S_HBT_2(self):
+    """Array Request: Heartbeat request from 3 CBSDs for GRANT renewal.
+
+    Returns response code 0.
+    """
+    # Register three devices.
+    registration_request = []
+    for device_filename in ('device_a.json', 'device_c.json', 'device_e.json'):
+      device = json.load(
+          open(os.path.join('testcases', 'testdata', device_filename)))
+      self._sas_admin.InjectFccId({'fccId': device['fccId']})
+      self._sas_admin.InjectUserId({'userId': device['userId']})
```

Comment:
Hello harinineelu,
'SasAdminImpl' object has no method 'InjectUserId'
I think you forgot to add it in 
-sas_interface.py
-sas.py

Reply:
Hi Assane, ''InjectUserId' has been added with PR #288. Since thats approved and merged, you should be able to see it in sas_interface.py now.